packages/@o3r/design/src/core/design-token/renderers/css/design-token-value.renderers.spec.ts

```diff
@@ -57,4 +57,32 @@ describe('getCssTokenValueRenderer', () => {
     expect(debug).toHaveBeenCalledWith(expect.stringContaining('var(--does-not-exist)'));
     expect(result).toBe('var(--does-not-exist)');
   });
+
+  describe('with extension value override', () => {
+    test('should not override non-numeric value', () => {
+      const renderer = getCssTokenValueRenderer();
+      const variable = designTokens.get('example.var-color-unit-ratio-override');
+
+      const result = renderer(variable, designTokens);
+      expect(result).toBe('#000');
+    });
+
+    test('should override numeric value and add unit', () => {
+      const renderer = getCssTokenValueRenderer();
+      const variable = designTokens.get('example.var-number-unit-ratio-override');
+
+      const result = renderer(variable, designTokens);
+      expect(result).not.toBe('2');
+      expect(result).toBe('5px');
```

Comment:
the first expect seems redundant
```suggestion
      expect(result).toBe('5px');
```

Reply:
a comment might be enough for that